src/templates/program-page.js

```diff
@@ -679,14 +669,10 @@ export const query = graphql`query ($id: String) {
                     field_link_image {
                       relationships {
                         field_media_image {
-                          localFile {
-                            publicURL
-                            childImageSharp {
-                              resize(width: 400, height: 300, cropFocus: CENTER) {
-                                src
-                              }
-                            }
-                          }
+                          gatsbyImage(
+                            width: 400
```

Comment:
Did we get rid of the height for a specific reason?

Reply:
No, that seems to be an oversight. Restored.